### PR TITLE
[zh-cn] Update json reference

### PR DESCRIPTION
**Pull request by jasonren0403**
### Related issues and pull requests

Follow up of #18437

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/json/index.md [hunk 1]
```diff
@@ -1,113 +1,150 @@
...
 ```
 
-在 `JSONNumber`（数字内部不允许包含空格）或 `JSONString`（字符串内部的空格被解释为相应的字符，否则就有问题了）之外的任何位置可以有多余的空白字符。JSON 只支持这些空白字符：制表符（[U+0009](https://symbl.cc/cn/0009/)），回车（[U+000D](https://symbl.cc/cn/000D/)），换行（[U+000A](https://symbl.cc/cn/0020/)）以及空格（[U+0020](https://symbl.cc/cn/0020/)）。
+无关紧要的{{Glossary("whitespace", "空白符")}}可以出现在任何地方，但不包括在 `JSONNumber`（数字不得包含空白）或 `JSONString`（字符串中的相应字符会被解释为空白，否则会导致错误）中。制表符（[U+0009](https://symbl.cc/cn/0009/)）、回车符（[U+000D](https://symbl.cc/cn/000D/)）、换行符（[U+000A](https://symbl.cc/cn/000A/)）和空格（[U+0020](https://symbl.cc/cn/0020/)）字符是唯一有效的空白字符。
+
+## 静态属性
 
-## 方法
+- `JSON[@@toStringTag]`
+  - : [`@@toStringTag`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag)属性的初始值为字符串 `"JSON"`。该属性在 {{jsxref("Object.prototype.toString()")}} 中使用。
```
Comment (skyclouds2001) [inline files/zh-cn/web/javascript/reference/global_objects/json/index.md:93]:
```suggestion
  - : [`@@toStringTag`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) 属性的初始值为字符串 `"JSON"`。该属性在 {{jsxref("Object.prototype.toString()")}} 中使用。
```

File: files/zh-cn/web/javascript/reference/global_objects/json/index.md [hunk 2]
```diff
@@ -1,113 +1,150 @@
...
 title: JSON
 slug: Web/JavaScript/Reference/Global_Objects/JSON
+l10n:
+  sourceCommit: b1f371983ebac0233e12adbde67a1d8c4f54563c
 ---
 
 {{JSRef}}
 
-**`JSON`** 对象包含两个方法：用于解析 [JavaScript Object Notation](https://json.org/)（{{glossary("JSON")}}）的 `parse()` 方法，以及将对象/值转换为 JSON 字符串的 `stringify()` 方法。除了这两个方法，JSON 这个对象本身并没有其他作用，也不能被调用或者作为构造函数调用。
+**`JSON`** 命名空间对象包含用于解析[JavaScript 对象表示法](https://json.org/)（{{glossary("JSON")}}）和将值转换为 JSON 字符串的静态方法。
```
Comment (skyclouds2001) [inline files/zh-cn/web/javascript/reference/global_objects/json/index.md:10]:
```suggestion
**`JSON`** 命名空间对象包含用于解析 [JavaScript 对象表示法](https://json.org/)（{{glossary("JSON")}}）和将值转换为 JSON 字符串的静态方法。
```

File: files/zh-cn/web/javascript/reference/global_objects/json/index.md [hunk 3]
```diff
@@ -1,113 +1,150 @@
...
+    escape (
+        %x22 /          ; "    引号     U+0022
+        %x5C /          ; \    反斜杠   U+005C
+        %x2F /          ; /    正斜杠   U+002F
+        %x62 /          ; b    退格符   U+0008
+        %x66 /          ; f    换页符   U+000C
+        %x6E /          ; n    换行符   U+000A
+        %x72 /          ; r    回车符   U+000D
+        %x74 /          ; t    制表符   U+0009
+        %x75 4HEXDIG )  ; uXXXX                U+XXXX
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/json/index.md:78]:
```suggestion
        %x22 /          ; "    引号     U+0022
        %x5C /          ; \    反斜杠   U+005C
        %x2F /          ; /    正斜杠   U+002F
        %x62 /          ; b    退格符   U+0008
        %x66 /          ; f    换页符   U+000C
        %x6E /          ; n    换行符   U+000A
        %x72 /          ; r    回车符   U+000D
        %x74 /          ; t    制表符   U+0009
        %x75 4HEXDIG )  ; uXXXX        U+XXXX
```

File: files/zh-cn/web/javascript/reference/global_objects/json/index.md [hunk 4]
```diff
@@ -1,113 +1,150 @@
...
-## 方法
+- `JSON[@@toStringTag]`
+  - : [`@@toStringTag`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) 属性的初始值为字符串 `"JSON"`。该属性在 {{jsxref("Object.prototype.toString()")}} 中使用。
+
+## 静态方法
 
 - {{jsxref("JSON.parse()")}}
   - : 解析 JSON 字符串并返回对应的值，可以额外传入一个转换函数，用来将生成的值和其属性，在返回之前进行某些修改。
 - {{jsxref("JSON.stringify()")}}
   - : 返回与指定值对应的 JSON 字符串，可以通过额外的参数，控制仅包含某些属性，或者以自定义方法来替换某些 key 对应的属性值。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/json/index.md:100]:
```suggestion
- {{jsxref("JSON.parse()")}}
  - : 解析 JSON 字符串并返回对应的值，可以额外传入一个转换函数，用来将生成的值和其属性，在返回之前进行某些修改。
- {{jsxref("JSON.stringify()")}}
  - : 返回与指定值对应的 JSON 字符串，可以通过额外的参数，控制仅包含某些属性，或者以自定义方法来替换某些属性值。
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions.

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Thank you! @jasonren0403

**Pull request merged**